guillainbisimwa/web3-journey
# Initial Webpage and Smart Contract Setup

## Overview
This pull request introduces the initial setup for building a decentralized application (dApp) on Ethereum. The key components include an HTML file (`index.html`), a smart contract, and the integration of ethers.js as an external library.

## Changes Made
1. **Webpage Construction:**
   - Added a new file: `index.html` to kickstart the webpage development.

2. **Smart Contract Development:**
   - Introduced a smart contract to facilitate decentralized functionality.

3. **External Library Integration:**
   - Included ethers.js as an external library to support Ethereum-related functionality.

4. **Version Compatibility Fix:**
   - Resolved version issues related to dependencies.

## How to Test
- Ensure the webpage loads successfully.
- Interact with the smart contract to verify its functionality.
- Confirm that ethers.js is properly integrated and functional.
- Check for any issues related to version compatibility.

## Additional Notes
This PR lays the groundwork for developing a decentralized application on the Ethereum blockchain. Further enhancements and feature additions can be built upon this foundation.

Please review and merge when ready.

GB
